com.civicrm.hrjobroles/tests/phpunit/CRM/Hrjobroles/BAO/HrJobRolesTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+require_once 'HrJobRolesTestBase.php';
+
+class CRM_Hrjobroles_BAO_HrJobRolesTest extends HrJobRolesTestBase {
+
+
+  function setUp() {
+    parent::setUp();
+  }
+
+  function tearDown() {
+    parent::tearDown();
+  }
+
+  function testCreateJobRoleWithBasicData() {
+    // create contact
+    $contactParams = array('first_name'=>'walter', 'last_name'=>'white');
+    $contactID = $this->individualCreate($contactParams);
+    // create contract
+    $contract = $this->createJobContract($contactID, date('Y-m-d', strtotime('-14 days')));
+    // create role
+    $roleParams = array(
+      'job_contract_id' => $contract->id,
+      'title' => 'test role'
+    );
+    $jobRole = $this->createJobRole($roleParams);
+    $this->assertNotEquals(NULL, $jobRole);
```

Comment:
hmmmm.... I don't think this is a good assertion. If createJobRole return anything that is not null (a Case, for example) the test will still pass. This would be better here:
- Assert that createJobRole return a class with an ID
- Load the job role with the returned ID
- Assert that the loaded job roles has the same title as the one you created